util/postman/openkit.postman_collection.json

```diff
@@ -117,6 +117,46 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "refresh token",
```

Comment:
let's make it clear that it's refreshing the user-scoped token. For example, name it: "user refresh token" or "passwordless refresh token"?